build/charts/flow-aggregator/templates/rolebindings.yaml

```diff
@@ -0,0 +1,48 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    app: flow-aggregator
+  name: flow-exporter-role-binding
+  namespace: {{ .Release.Namespace }}
+subjects:
+- kind: ServiceAccount
+  name: antrea-agent
+  namespace: kube-system
```

Comment:
Is there a way to set this dynamically?  considering Antrea can be installed in a different NS.

Reply:
> Is there a way to set this dynamically? considering Antrea can be installed in a different NS.

just to clarify, this was not introduced by this PR, but yes I can address it here

> maybe add a variable - antreaNamespace?

that's what I was thinking too